screens/Auth/AddGuestNext.js

```diff
@@ -64,8 +118,11 @@ const AddGuestNext = () => {
 
     console.log("Payload being sent to the server:", JSON.stringify(payload));
```

Comment:
This shouldn't be logged inside the console. username, password, and guest details would be exposed